hermes-console-vue/src/store/consistency/useConsistencyStore.ts

```diff
@@ -0,0 +1,71 @@
+import { defineStore } from 'pinia';
+import {
+  fetchConsistencyGroups,
+  fetchInconsistentGroups,
+} from '@/api/hermes-client';
+import type { ConsistencyStoreState } from '@/store/consistency/types';
+import type {
+  InconsistentGroup,
+  InconsistentTopic,
+} from '@/api/inconsistent-group';
+
+const partition = (arr: string[], batch_size: number): string[][] => {
```

Comment:
I'd say that this would be misleading since this is a generic helper function which can accept any array of string. I've changed the argument name to `array`